includes/parser/Parser.php

```diff
@@ -5710,7 +5710,12 @@ private function extractSections( $text, $sectionId, $mode, $newText = '' ) {
 		if ( !$node ) {
 			# Not found
 			if ( $mode === 'get' ) {
```

Comment:
Can we change this condition into and prevent nesting if conditions?

```suggestion
			if ( $mode === 'get' && $newText ) {
```